manifests/r/Remeha/ServiceToolRemeha/7.0.53.0/Remeha.ServiceToolRemeha.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Remeha.ServiceToolRemeha

```

Comment:
Should this package be identified as `Remeha.Recom`?

![image](https://github.com/microsoft/winget-pkgs/assets/114952565/457ee036-e4f5-4132-ac15-34ef63e72e17)

> Description: Recom is a software program developed by Remeha B.V. It offers the possibility to communicate directly to Remeha boilers/appliances.